solutions/usaco-526.mdx

```diff
@@ -13,6 +13,30 @@ author: Brad Ma
 
 <LanguageSection>
 
+<PySection>
+
+```python
+import sys
+sys.stdin = open("censor.in", "r")
+sys.stdout = open("censor.out", "w")
+
+def main():
+	s = input().strip()
+	t = input().strip()
+	final = ""
+	# Add each character to the final string
+	for char in s:
+		final += char
+		# If the last len(t) characters is just t, then remove the last len(t) characters
+		if len(final) >= len(t) and final[-len(t):] == t:
+			final = final[:-len(t)]
+	return final
+
+print(main())
```

Comment:
take this out of main (unless it TLEs without)

Reply:
Okay, I made that change.